tests/test_bpo.py

```diff
@@ -291,3 +291,66 @@ def __init__(self, *, getitem=None):
     await bpo.router.dispatch(event, gh)
     assert gh.patch_data is None
     assert gh.patch_url is None
+
+
+@pytest.mark.asyncio
+async def test_set_comment_body_success():
+    data = {
+        "action": "opened",
+        "pull_request": {
+            "title": "[3.6] bpo-12345: some issue",
+            "url": "https://api.github.com/repos/python/cpython/pulls/4321",
+            "statuses_url": "https://api.github.com/blah/blah/git-sha",
+            "issue_url": "https://api.github.com/repos/blah/blah/issues/comments/123456",
+            "body": "An issue bpo-12345 in the body"
+        }
+    }
+        
+    event = sansio.Event(data, event="pull_request", delivery_id="123123")
+    gh = FakeGH()
+    await bpo.router.dispatch(event, gh)
+    body_data = gh.patch_data
+    assert "[bpo-12345](https://www.bugs.python.org/issue12345)" in body_data["body"]
+    assert "123456" in gh.patch_url
+
+
+@pytest.mark.asyncio
+async def test_set_comment_body_without_bpo():
+    data = {
+        "action": "created",
+        "comment": {
+            "url": "https://api.github.com/repos/blah/blah/issues/comments/123456",
+            "body": "This body doesn't contain any bpo text"
+        }
+    }
+
+    event = sansio.Event(data, event="issue_comment", delivery_id="123123")
+    gh = FakeGH()
+    await bpo.router.dispatch(event, gh)
+    assert gh.patch_data is None
+    assert gh.patch_url is None
+
+
+@pytest.mark.asyncio
+async def test_set_comment_body_already_hyperlink_bpo():
+    data = {
+        "action": "opened",
+        "pull_request": {
+            "title": "[3.6] bpo-12345: some issue",
+            "url": "https://api.github.com/repos/python/cpython/pulls/4321",
+            "statuses_url": "https://api.github.com/blah/blah/git-sha",
+            "issue_url": "https://api.github.com/repos/blah/blah/issues/comments/123456",
+            "body": ("bpo-123"
+                    "[bpo-123](https://www.bugs.python.org/issue123)"
+                    "<a href='https://www.bugs.python.org/issue123'>bpo-123</a>"
+                   )
+                   
+        }
+    }
+    
+    event = sansio.Event(data, event="pull_request", delivery_id="123123")
+    gh = FakeGH()
+    await bpo.router.dispatch(event, gh)
+    body_data = gh.patch_data
+    assert body_data["body"].count("[bpo-123](https://www.bugs.python.org/issue123)") == 2
+    assert '123456' in gh.patch_url
```

Comment:
Please use double quotes to be consistent with the rest.

Reply:
Really sorry for this error. I fixed this, and there was one previous occurence, so I fixed that too.
If the change for previous occurence of single quotes shouldn't have been done, I'll be happy to revert that change :slightly_smiling_face: